lib/index.js

```diff
@@ -2,9 +2,11 @@
  * @author Toru Nagashima <https://github.com/mysticatea>
  * See LICENSE file in root directory for full license.
  */
-"use strict";
+import { readFileSync } from "fs";
 
-const KEYS = require("./visitor-keys.json");
+const KEYS = JSON.parse(
+    readFileSync(new URL("./visitor-keys.json", import.meta.url))
```

Comment:
This doesn't work in `dist/eslint-visitor-keys.cjs`, it tries to load `dist/visitor-keys.json`.

Also, `eslint-visitor-keys` is used by `espree` and ESLint's `Linter`, which are intended to work in browsers as well (e.g., in our [Online Demo](https://eslint.org/demo)). Even if we manage to fix the issue with `dist/eslint-visitor-keys.cjs`, I'm not sure if bundlers used in projects that depend on `eslint-visitor-keys` will be able to properly handle this.

Thoughts about converting `visitor-keys.json` to `visitor-keys.js` module? That would avoid any possible issues with loading local files.

Reply:
SGTM, esp. if there is deemed to be no need to export that JSON file as discussed previously.

Reply:
`visitor-keys.json` wasn't documented in README, so I think it's fine to remove it in a major version.

@nzakas thoughts about this?

Reply:
Agree

Reply:
Ok, switched to a module.